packages/content-type-reply/README.md

```diff
@@ -0,0 +1,94 @@
+# Reply content type
+
+This package provides an XMTP content type to support direct replies to messages.
+
+## What’s a reply?
+
+A reply action is a way to respond directly to a specific message in a conversation. Instead of sending a new message, users can select and reply to a particular message.
+
+## Why replies?
+
+Providing replies in your app enables users to maintain context and clarity in their conversations. Replies can also help organize messages, making messages easier to find and reference in the future. This user experience can help make your app a great tool for collaboration.
+
+## Install the package
+
+```bash
+# npm
+npm i @xmtp/content-type-reply
+
+# yarn
+yarn add @xmtp/content-type-reply
+
+# pnpm
+pnpm i @xmtp/content-type-reply
+```
+
+## Create a reply
+
+With XMTP, replies are represented as objects with the following keys:
+
+- `reference`: The message ID for the message that is being reacted to
+- `content`: A string representation of the reply
+
+```tsx
+const reply: Reply = {
+  reference: someMessageID,
+  content: "I concur",
+};
+```
+
+## Send a reply
+
+Now that you have a reply, you can send it:
+
+```tsx
+await conversation.messages.send(reply, {
+  contentType: ContentTypeReaction,
```

Comment:
Just curious @rygine  - should this be `ContentTypeReply`?

Reply:
yep! this was mostly a copy/paste of the reaction type and looks like something got passed my find/replace. good catch!